CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+* `compile`: Support **arm64** cross compiler using docker; support **Ububtu 14.04+**, **CentOS7.6+**. [#779](https://github.com/chubaofs/chubaofs/pull/779)
+* `client`: Enable read from nearest **DataNode** IP. [#810](https://github.com/chubaofs/chubaofs/pull/810)  
+* `client`: Update volume follower read config from master periodicity. [#837](https://github.com/chubaofs/chubaofs/pull/837)  
+
+### Refactor
+* `object`: listObject and listObjectsV2 list result is not expected. [#769](https://github.com/chubaofs/chubaofs/pull/769)
+* `master`: Check whether the used space between the replicas is consistent when performing the load partition operation. [#813](https://github.com/chubaofs/chubaofs/pull/813)
+* `metanode` `datanode`: Optimize batch delete Extent; add **autoRepairLimitRater** on **DataNode**. [#781](https://github.com/chubaofs/chubaofs/pull/781)
+* `datanode`: Add **autoRepair** limit on **DataNode**. [#842](https://github.com/chubaofs/chubaofs/pull/842)  
+* `datanode`: **repairTask** skip the extent which has been deleted. [#842](https://github.com/chubaofs/chubaofs/pull/842) 
+* `security`: Add security disclosure process. [#782](https://github.com/chubaofs/chubaofs/pull/782) 
+
+### Bug fix
+* `master`: When loading metadata of cluster, using the current ID instead of the loaded ID. [#821](https://github.com/chubaofs/chubaofs/pull/821) 
+* `metanode`: **MetaNode** delete **dEntry** deadlock. [#785](https://github.com/chubaofs/chubaofs/pull/785) 
+* `metanode`: Fixes [#760](https://github.com/chubaofs/chubaofs/issues/760) Add **inode** to freeList when NLink is 0, and delete **inode** 7 days later. [#767](https://github.com/chubaofs/chubaofs/issues/767)
+* `metanode`: Add EXTENT_DEL file extentKey header and checksum verify. [#813](https://github.com/chubaofs/chubaofs/pull/813)  
+* `datanode` `metanode`: When **DataNode** and **MetaNode** start, the partitions not exist in cluster view will be renamed to **expiredPartition** and skip loading. [#824](https://github.com/chubaofs/chubaofs/pull/824) 
+* `datanode` `metanode`: Fixes issue[#698](https://github.com/chubaofs/chubaofs/issues/698), Raft instance delete itself by applying ConfChange raft log. [#866](https://github.com/chubaofs/chubaofs/pull/866)  
+* `metanode`: **MetaNode** may not free space. [#838](https://github.com/chubaofs/chubaofs/pull/838)  
+* `client`: Refresh Extents cache when opening file. [#783](https://github.com/chubaofs/chubaofs/pull/783) 
+* `client`: Batch inode get mechanism is out of service. [#847](https://github.com/chubaofs/chubaofs/pull/847) 
+* `client`: Mount failure for authorized user. [#828](https://github.com/chubaofs/chubaofs/pull/828) 
+* `datanode`: Several bugs in automatic repair for **tinyExtent**. [#855](https://github.com/chubaofs/chubaofs/pull/855) [#857](https://github.com/chubaofs/chubaofs/pull/857) 
```

Comment:
It it recommended to modify to "`Fix serval issue in automatic repair process for tiny extent`".